syntax = "proto3";
package fixbug;

option cc_generic_services = true;

message ResultCode
{
    int32 errnum = 1;
    bytes errmsg = 2;
}

message LoginRequest
{
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse
{
    ResultCode rcode = 1;
    bool success = 2;
}

service UserService
{
    rpc login(LoginRequest) returns(LoginResponse);
}